atomic_reactor/plugins/post_push_to_pulp.py

```diff
@@ -206,8 +207,18 @@ def push_tarball_to_pulp(self, image_names, repo_prefix="redhat-"):
                 p.upload(outfile.name)
         except:
             self.log.debug("Error on creating deduplicated layers tar", exc_info=True)
-            self.log.info("Falling back to full tar upload")
-            p.upload(self.filename)
+            try:
+                if file_extension != '.tar':
+                    raise RuntimeError("tar is already compressed")
+                with NamedTemporaryFile(prefix='full_tar_', delete=True) as outfile:
```

Comment:
Right, I just though it would make sense to make it explicit (it actually doesn't). Removed in next commit